lerna.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Should we use `ignoreChanges` to prevent publishing when certain files are changed? E.g., README, tests, etc.

Reply:
Yep, you may check how we are doing in pico. It’s not that important for packages we won’t be working daily, but you may add it nevertheless.

Reply:
Done ✅